requirements.txt

```diff
@@ -5,35 +5,41 @@
 #    pip-compile --output-file=requirements.txt requirements.in
 #
 contourpy==1.0.6
-    # via matplotlib
+    # via
+    #   -r requirements.in
```

Comment:
I am unsure as to why do you need requirements.in file for this assignment..I am assuming you are preparing this script for Final Project :)